mcs/class/System/System.Net.NetworkInformation/Win32NetworkInterfaceMarshal.cs

```diff
@@ -34,6 +34,9 @@ class Win32NetworkInterface {
 		[DllImport ("iphlpapi.dll", SetLastError = true)]
 		static extern int GetNetworkParams (IntPtr ptr, ref int size);
 
+		[DllImport("kernel32.dll", EntryPoint="MultiByteToWideChar")]
```

Comment:
This likely needs `SetLastError = true` as in the call above. 

And nit if you are making that change, I don't think this needs `EntryPoint`.